zerver/lib/onboarding_steps.py

```diff
@@ -52,6 +52,9 @@ def to_dict(self) -> dict[str, str]:
     OneTimeNotice(
         name="interleaved_view_messages_fading",
     ),
+    OneTimeNotice(
+        name="first_topic_resolved",
```

Comment:
"first_time_topic_resolved" would be better or simply "topic_resolved" as it is already an attribute of `OneTimeNotice`?

I'm feeling "first_topic_resolved" a bit confusing, like: first topic of a stream resolved or something.

Reply:
lets then go with `topic_resolved` ?